src/modules/material-requests/types/index.ts

```diff
@@ -31,6 +31,7 @@ export enum MaterialRequestType {
 	REUSE = 'REUSE',
 	MORE_INFO = 'MORE_INFO',
 	VIEW = 'VIEW',
+	ALL = 'ALL',
```

Comment:
Not sure if we need to add the `ALL` option here since it doesn't exist in the data model.